commands/raids/leave.js

```diff
@@ -4,36 +4,41 @@ const Commando = require('discord.js-commando');
 const Raid = require('../../app/raid');
 
 class LeaveCommand extends Commando.Command {
-	constructor(client) {
-		super(client, {
-			name: 'leave',
-			group: 'raids',
-			memberName: 'leave',
-			description: 'Can\'t make it to a raid? no problem, just leave it.'
-		});
-	}
-
-	run(message, args) {
-		var params = args.split(' ');
-		var raid_id = params[0];
-		var info = {};
-
-		if (!raid_id) {
-			message.reply('Please enter a raid id which can be found on the raid post.  If you do not know the id you can ask for a list of raids in your area via `!status`.');
-			return;
-		}
-
-		info = Raid.removeAttendee(message.channel, message.member, raid_id);
-
-		if (!info.error) {
-			message.react('👍');
-			// message.member.send(`You have left raid **${info.raid.id}**.`);
-
-			// get previous bot message & update
-			Raid.getMessage(message.channel, message.member, info.raid.id)
-				.edit(Raid.getFormattedMessage(info.raid));
-		}
-	}
+  constructor(client) {
+    super(client, {
+      name: 'leave',
+      group: 'raids',
+      memberName: 'leave',
+      aliases: ['part'],
+      description: 'Can\'t make it to a raid? no problem, just leave it.',
+      argsType: 'multiple'
+    });
+  }
+
+  run(message, args) {
+    if (message.channel.type !== 'text') {
+      message.reply('Please leave a raid from a public channel.');
+      return;
+    }
+
+    const raid = Raid.findRaid(message.channel, message.member, args);
```

Comment:
this is 100% personal preference, I'm not expecting a change or anything.  I tend to not call something by a variable if that's not actually what it is.  So like below it's `raid.raid.id` so clearly this isn't the typical "raid" data, so a normal `raid.id` would not work b/c it's not actually a raid.  I end up calling this "info" or something generic so that I don't get confused later wondering why "raid.id" isn't working haha.